src/net_processing.cpp

```diff
@@ -1926,6 +1926,19 @@ bool static ProcessMessage(CNode* pfrom, const std::string& strCommand, CDataStr
         return true;
     }
 
+    // VeriBlock: if POP is not enabled, ignore POP-related P2P calls
+    if (VeriBlock::isPopActive()) {
+        auto& state = *State(pfrom->GetId());
```

Comment:
What is the lifetime of this? Want to make sure this is not a temporary reference.

Reply:
Updated